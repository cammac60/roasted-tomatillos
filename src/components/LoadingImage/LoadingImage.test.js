import React from 'react';
import { shallow } from 'enzyme';
import LoadingImage from './LoadingImage';

it("should match snapshot with all data passed in correctly", () => {
  const image = shallow(<LoadingImage />);
  expect(image).toMatchSnapshot();
});
