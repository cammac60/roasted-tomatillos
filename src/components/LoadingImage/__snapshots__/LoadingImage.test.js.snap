// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with all data passed in correctly 1`] = `
<div
  className="loading"
>
  <img
    alt="camera"
    src="camera.svg"
  />
  <div
    className="light"
  />
  <img
    alt="film"
    className="movie-circle"
    src="film-reel.svg"
  />
</div>
`;
